boards/tutorials/nrf52840dk-dynamic-apps-and-policies/README.md

```diff
@@ -0,0 +1,10 @@
+nRF52840DK Board Definition for the Tock Dynamic Apps and Secure Policies Tutorial
+==================================================================================
+
+This is the board definition for the nRF52840DK target used in the
+[Dynamic App Loading and Secure Policies]
+(https://github.com/tock/book/pull/60/files).
```

Comment:
This is an unfortunate chicken-and-egg situation; fine for the moment, just make sure you have a note somewhere to update this when https://github.com/tock/book/pull/60 lands

(non-blocking comment)